component/core/src/main/java/org/exoplatform/social/core/space/SpaceUtils.java

```diff
@@ -2033,4 +2036,16 @@ public static ResourceBundleService getResourceBundleService() {
 
     return trimmed.toArray(new String[trimmed.size()]);
   }
+
+  private static void clearIdentityCaching(String username) {
+    ExoContainer container = ExoContainerContext.getCurrentContainer();
+
+    IdentityManager identityManager = container.getComponentInstanceOfType(IdentityManager.class);
+    CachedIdentityStorage cachedIdentityStorage = container.getComponentInstanceOfType(CachedIdentityStorage.class);
+
+    Identity userIdentity = identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, username);
```

Comment:
Could you make org.exoplatform.social.core.storage.cache.CachedIdentityStorage#clearIdentityCached get a Long as ID instead of the whole Identity. This will help avoid reloading the identity here.